<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connectionManager</class>
 <widget class="QWidget" name="connectionManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>250</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>557</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>557</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="l_interval">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Interval (ms):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="t_interval">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>133</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>30</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="b_start_sending">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Start telemetry</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="b_stop_sending">
        <property name="minimumSize">
         <size>
          <width>129</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>129</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Stop telemetry</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
